app/controllers/finance/banding_tracker/provider_choices_controller.rb

```diff
@@ -11,7 +11,11 @@ def new
       def create
         choose_provider_form.attributes = choose_provider_params
 
-        return redirect_to finance_banding_tracker_provider_path(id: choose_provider_form.id) if choose_provider_form.valid?
+        if choose_provider_form.valid?
+          return redirect_to finance_banding_tracker_provider_path(id: choose_provider_form.id)
```

Comment:
It was like this already but it might be a bit neater to move the `@providers =... render :new` into the else so we don't have to return. It's match more closely with the other controller actions that way.

Low priority of course.